basics/getting_started/plugin_compatibility.md

```diff
@@ -21,19 +18,22 @@ Underlying those shared features are shared components.
 When authoring a plugin for the IntelliJ Platform, it is important to understand and declare dependencies on these components. 
 Otherwise, it may not be possible to load or run the plugin in a product because the components on which it depends aren't available.
 
+* bullet list
+{:toc}
+
 ## Declaring Plugin Dependencies
 For the purposes of dependencies, a _module_ can be thought of like a built-in plugin that ships as a non-removable part of a product. 
 A working definition of a dependency is that a plugin project cannot be run without the module present in an IntelliJ Platform-based product. 
 Declaring a dependency on a module also expresses a plugin's compatibility with a product in that the IntelliJ Platform determines whether a product contains the correct modules to support a plugin before loading it. 
 
-Note:
-* If a plugin does not include any module dependency tags in its `plugin.xml` file, it's assumed to be a legacy plugin and is loaded only in IntelliJ IDEA.
-  This form of dependency "non-declaration" is deprecated; do not use it for new plugin projects. 
-* If the `plugin.xml` file includes one or more dependency tags, the plugin is only loaded by the IntelliJ Platform if the product contains _all the modules_ on which the plugin depends.
-
 [Part I](/basics/plugin_structure/plugin_dependencies.md) of this document describes the syntax for declaring plugin dependencies and optional plugin dependencies.  
 Part II of this document (below) describes the functionality of the IntelliJ Platform modules to aid in determining the dependencies of a plugin. 
 
+The way dependency declarations are handled by the Intellij Platform is determined by the contents of the `plugin.xml` file:
+* If a plugin does not declare any dependencies in its `plugin.xml` file, or if it declares dependencies only on other plugins but not modules, it's assumed to be a legacy plugin and is loaded only in IntelliJ IDEA.
+  This configuration of dependency declaration is deprecated; do not use it for new plugin projects. 
+* If a plugin declares at least _one_ module dependency in its `plugin.xml` file, the plugin is loaded if a JetBrains' IntelliJ Platform-based product contains _all the modules and plugins_ on which the plugin has declared a dependency.
```

Comment:
I would replace "JetBrains' IntelliJ Platform-based" by "IntelliJ Platform-based" because this logic will work for third-party IDEs as well.